app/views/workarea/storefront/searches/autocomplete.html.haml

```diff
@@ -1,31 +1,32 @@
 .grid.grid--rev
-  .grid__cell.grid__cell--25
+  .grid__cell.grid__cell--25-at-medium.grid__cell--100-at-small
     - if @autocomplete.searches.any?
-      %span.search-autocomplete__heading
-        - if @autocomplete.trending_searches?
-          = t('workarea.storefront.search_autocomplete.trending_searches')
-        - else
-          = t('workarea.storefront.search_autocomplete.popular_searches')
+      .search-autocomplete__searches
```

Comment:
I changed the hierarchy of the `.search-autocomplete__searches` element here in order to bring parity to the `.search-autocomplete__products` element. The heading was previously located outside the element, which prevented clean CSS and would have required knowledge of the outer containing element.